### PR TITLE
Upgrade to quiver ^2.0.0

### PR DESCRIPTION
**Pull request by Elvis5566**
## Motivation
quiver is out of date, conflict with other libs, in my case, [jaguar_orm](https://pub.dev/packages/jaguar_orm)

## Changes
Change quiver version to ^2.0.0.

#### Release Notes
Change quiver version to ^2.0.0.

@Workiva/app-frameworks

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: pubspec.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ dependencies:
   w_common: ^1.13.0
   w_flux: ^2.9.5
   platform_detect: ^1.3.4
-  quiver: ">=0.25.0 <1.0.0"
+  quiver: "^2.0.0"
```
Comment (greglittlefield-wf) [inline pubspec.yaml]: Thanks for the PR! Would you be opposed to leaving the lower bound as-is? It seems we still have some internal dependencies on old versions that wouldn't resolve with this range... 😓 
```suggestion
  quiver: ">=0.25.0 <3.0.0"
```

**Comment by greglittlefield-wf** [comment]
Merging, thanks!!

+10

@Workiva/release-management-pp

**Comment by greglittlefield-wf** [comment]
I'm signing off for the day, but we should be able to get this released tomorrow morning!

**Comment by greglittlefield-wf** [comment]
Published as over_react `2.5.2+dart2`

**Pull request merged**